orchagent/portsorch.cpp

```diff
@@ -61,6 +61,9 @@ extern string gMyAsicName;
 #define DEFAULT_VLAN_ID     1
 #define MAX_VALID_VLAN_ID   4094
 
+#define DEFAULT_VECTOR_SIZE 16
+
+#define PORT_STAT_POLLING_SEC                             5
```

Comment:
Please rename this variable as requested in the other PR

Reply:
Done